File: src/ScenePackeage/SceneAdd.java
package ScenePackeage;

import com.alibaba.fastjson.JSON;
import database.movieSystem.MovieSystemDB;
import database.sceneUtil.Scene;
import frontEnd.utils.ServletUtils;

import javax.servlet.ServletException;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import java.io.IOException;

public class SceneAdd extends HttpServlet {
    @Override
    public void init() throws ServletException {
        super.init();
    }

    @Override
    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
        doGet(req, resp);
    }

    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
        System.out.println("接收到请求");
        String Sno = req.getParameter("Sno");
        String Mno=  req.getParameter("Mno");
        String Tno=  req.getParameter("Tno");
        String Tbrand=req.getParameter("Tbrand");
        String beginTime=req.getParameter("beginTime");
        String endTime=req.getParameter("endTime");
        String language=req.getParameter("language");
        String roomType=req.getParameter("roomType");
        String roomName=req.getParameter("roomName");
        String location=req.getParameter("location");
        String Sdate=req.getParameter("Sdate");
        String price=req.getParameter("Price");
        resp.setContentType("application/json");
        resp.setCharacterEncoding("UTF-8");

        //添加数据
        Scene scene=new Scene(Sno,Mno,Tno,Tbrand,beginTime,endTime,language,roomType,roomName,location,Sdate,"",Double.valueOf(price));
        boolean flag= MovieSystemDB.getSceneTable().insert(scene);
        //传回添加标志
        ServletUtils.resJsonString(resp, JSON.toJSONString(flag));

    }
}

File: web/js/scene-add.js
$(document).ready(function () {
    //页面加载请求数据加载Option
    $("#Scene-add-button").click(function () {
        debugger;
        if($("#beginTime").val()>$("#endTime").val()){
            alert("格式出错，请重新输入");
            $("input").html("");
        }
        else {
            $.ajax({
                type: "POST",
                url: "/SceneAdd",
                async: false,
                data: {
                    "Sno": $("#Sno").val(),
                    "Mno": $("#Mno").val(),
                    "Tno": $("#Tno").val(),
                    "Tbrand": $("#Tbrand").val(),
                    "beginTime": $("#beginTime").val(),
                    "endTime": $("#endTime").val(),
                    "language": $("#language").val(),
                    "roomType": $("#roomType").val(),
                    "roomName": $("#roomName").val(),
                    "location": $("#location").val(),
                    "Sdate": $("#location").val(),
                    "Price": $("#Price").val()
                },
                dataType: "json",
                success: function (data) {
                    debugger;
                    if (data === false) {
                        alert("传送失败")
                    } else if (data === true) {
                        alert("状态：插入成功");
                        // alert("data:" + data);
                        Myclose();
                    }
                },
                error: function () {
                    alert("传送失败")
                }
            })
        }
        function  Myclose() {
            window.opener=null;window.open('','_self');window.close();
        }
    })
});